### PR TITLE
[ra2] proofread ch5 and ch7

### PR DESCRIPTION
**Pull request by rgstori**
fixes [#3458](https://github.com/anuket-project/anuket-specifications/issues/3458)

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter05.rst [hunk 1]
```diff
@@ -4,198 +4,155 @@ Security Guidance
...
+- Deploy applications only from trusted registries.
+- Secure Kubernetes container orchestration with administrative boundaries between tenants:
 
-   -  Use Namespaces to establish security boundaries between tenants.
-   -  Create and define Cluster network policies.
-   -  Run a Cluster-wide pod security policy.
-   -  Turn on Audit Logging.
-   -  Separate sensitive workloads using Namespaces.
-   -  Secure tenant metadata access.
+   - Use namespaces to establish security boundaries between tenants.
```
Comment (CsatariGergely) [inline doc/ref_arch/kubernetes/chapters/chapter05.rst:65]:
```suggestion
   - Use Namespaces to establish security boundaries between tenants.
```

Namespaces here are referring to the Kubernetes object

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_arch/kubernetes/chapters/chapter05.rst [hunk 2]
```diff
@@ -4,198 +4,155 @@ Security Guidance
...
+   - Create and define cluster network policies.
```
Comment (CsatariGergely) [inline doc/ref_arch/kubernetes/chapters/chapter05.rst:66]:
```suggestion
   - Create and define cluster Network Policies.
```

File: doc/ref_arch/kubernetes/chapters/chapter05.rst [hunk 3]
```diff
@@ -4,198 +4,155 @@ Security Guidance
...
+   - Implement cluster-wide pod security policies.
+   - Enable audit logging.
+   - Isolate sensitive workloads using namespaces.
```
Comment (CsatariGergely) [inline doc/ref_arch/kubernetes/chapters/chapter05.rst:69]:
```suggestion
   - Isolate sensitive workloads using Namespaces.
```

File: doc/ref_arch/kubernetes/chapters/chapter05.rst [hunk 4]
```diff
@@ -4,198 +4,155 @@ Security Guidance
...
+Vulnerability Assessment
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Vulnerability assessments are a crucial part of IT risk management lifecycles.
-The mitigation of vulnerabilities helps in protecting systems and data from
-unauthorised access and breaches. Implement necessary vulnerability scanner tools,
-such as OpenVAS, or any other open-source or commercial tools, to identify threats
-and flaws within the infrastructure that represent potential risks.
+Regular vulnerability assessments are crucial for IT risk management. Implement vulnerability scanner tools (e.g.,
+OpenVAS, or other open-source or commercial tools) to identify and mitigate threats and vulnerabilities.
```
Comment (CsatariGergely) [inline doc/ref_arch/kubernetes/chapters/chapter05.rst:97]:
```suggestion
OpenVAS, or other open source or commercial tools) to identify and mitigate threats and vulnerabilities.
```

File: doc/ref_arch/kubernetes/chapters/chapter07.rst [hunk 5]
```diff
@@ -4,158 +4,193 @@ Gaps, Innovation, and Development
 Introduction to Gaps, Innovation, and Development
 -------------------------------------------------
 
-There are functional gaps between the current state of technology available in open source and the requirements of this
-Reference Architecture or the Reference Model. This chapter highlights these gaps in detail and provides proposed
-solutions. As a result, various “upstream” community projects may be identified and will be targeted for development
-efforts.
-
-Gap template
+Functional gaps exist between available open-source
```
Comment (CsatariGergely) [inline doc/ref_arch/kubernetes/chapters/chapter07.rst:7]:
```suggestion
Functional gaps exist between available open source
```

File: doc/ref_arch/kubernetes/chapters/chapter07.rst [hunk 6]
```diff
@@ -4,158 +4,193 @@ Gaps, Innovation, and Development
...
+**Baseline project:** *Kubernetes*
+
+**Gap description:**  L3 VRFs/VPNs are commonly used for
+traffic separation (signaling, charging, LI, O&M).  CNFs
+must interoperate with existing network elements,
+requiring Kubernetes Pod connection to L3 VPNs (currently
+only possible via Multus).  However, network orchestration
+(connecting the interface to a gateway router terminating
+the L3 VPN) isn't handled by Kubernetes, and lacks a
+production-grade open-source solution.
```
Comment (CsatariGergely) [inline doc/ref_arch/kubernetes/chapters/chapter07.rst:156]:
```suggestion
production-grade open source solution.
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_arch/kubernetes/chapters/chapter05.rst [hunk 7]
```diff
@@ -4,198 +4,155 @@ Security Guidance
...
 
-Patch management
+Patch Management
 ~~~~~~~~~~~~~~~~
 
-Patch management is another key aspect of the IT risk management lifecycle for
-security, compliance, uptime, feature enhancements, and so on. Implement the
-necessary patch management to ensure your environment is not susceptible to
-exploitation.
+Implement a robust patch management process to address security vulnerabilities, ensure compliance, maintain uptime,
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter05.rst:102]:
```suggestion
A robust patch management process should be implemented to address security vulnerabilities, ensure compliance,
```

File: doc/ref_arch/kubernetes/chapters/chapter05.rst [hunk 8]
```diff
@@ -4,198 +4,155 @@ Security Guidance
...
+and enable feature enhancements.
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter05.rst:103]:
```suggestion
maintain uptime, and enable feature enhancements.
```

File: doc/ref_arch/kubernetes/chapters/chapter05.rst [hunk 9]
```diff
@@ -4,198 +4,155 @@
...
+- Enforce access control on registries using unique credentials to limit who can control builds or add images.
+- Use TLS for all network access.
+- Configure user roles and access levels to ensure segregation of duties.
+
+   - Do not co-locate container and non-container services on the same node.
+   - Do not run containers as root.
+
+- Implement multi-factor authentication for all administrative access.
+- Harden the configuration using Center for Internet Security (CIS) benchmarks for container runtimes and Kubernetes.
+- Deploy security products providing allowlisting, behavior monitoring, and anomaly detection to prevent malicious
```
[the remark this answers is not in the document]
Comment (rgstori) [reply doc/ref_arch/kubernetes/chapters/chapter05.rst:126]: @CsatariGergely can you please allowlist allowlisting?

File: doc/ref_arch/kubernetes/chapters/chapter07.rst [hunk 10]
```diff
@@ -4,158 +4,193 @@ Gaps, Innovation, and Development
...
+
+**Baseline project:**  Identify the upstream project
+(e.g., *Kubernetes*) where the gap exists. If none, state
+"none".
 
-..
+**Gap description:** Describe missing functionality from
+the related requirements in known implementations. Include
+references to ongoing work in the target project that may
+address the gap.
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter07.rst:28]:
```suggestion
..
   **Gap description:** Describe missing functionality from
    the related requirements in known implementations. Include
    references to ongoing work in the target project that may
    address the gap.
```

File: doc/ref_arch/kubernetes/chapters/chapter07.rst [hunk 11]
```diff
@@ -4,158 +4,193 @@ Gaps, Innovation, and Development
...
 
-   **Related requirements:** List the requirement references ``abc.xyz.00`` from RA2 or RM which this gap tries to
-   address.
+**Related requirements:** List requirement references
+(e.g., ``abc.xyz.00``) from RA2 or RM addressed by this
+gap.
+
+**Baseline project:**  Identify the upstream project
+(e.g., *Kubernetes*) where the gap exists. If none, state
+"none".
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter07.rst:23]:
```suggestion
   **Baseline project:**  Identify the upstream project
   (e.g., *Kubernetes*) where the gap exists. If none, state
   "none".
```

File: doc/ref_arch/kubernetes/chapters/chapter07.rst [hunk 12]
```diff
@@ -4,158 +4,193 @@ Gaps, Innovation, and Development
...
+of :cite:t:`refmodel`).
 
 **Baseline project:** *Kubernetes*
 
-**Gap description:** The Memory Manager was added in v1.21 as alpha feature. For details, see
-:ref:`chapters/chapter03:management of memory and huge pages resources`.
+**Gap description:** The Memory Manager (alpha feature in
+v1.21) is addressed in
+:ref:`chapters/chapter03:management of memory and huge
+pages resources`.
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter07.rst:175]:
```suggestion
:ref:`chapters/chapter03:management of memory and huge pages resources`.
```

**Review by petorre — APPROVED** [review]

**Pull request merged**